rewatch/src/build/clean.rs

```diff
@@ -346,7 +347,7 @@ pub fn clean(
         show_progress,
         // Build the package tree with dev dependencies.
         // They should always be cleaned if they are there.
-        true,
+        DevDeps::Clean,
```

Comment:
This used to be `true`, so `dev` files are included.
Yet, these files never existed in the first place because they were never shipped as part of npm package.